debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -1103,6 +1104,67 @@ public void shouldReceiveChangesForTableWithoutPrimaryKey() throws Exception{
         assertRecordSchemaAndValues(expectedAfter, deletedRecord, Envelope.FieldName.AFTER);
     }
 
+    @Test
+    @FixFor("DBZ-1146")
+    public void shouldReceiveChangesForReplicaIdentityFullTableWithToastedValue() throws Exception {
+        testReceiveChangesForReplicaIdentityFullTableWithToastedValue(SchemaRefreshMode.COLUMNS_DIFF_EXCLUDE_UNCHANGED_TOAST, true);
+        testReceiveChangesForReplicaIdentityFullTableWithToastedValue(SchemaRefreshMode.COLUMNS_DIFF_EXCLUDE_UNCHANGED_TOAST, false);
+    }
+
+    private void testReceiveChangesForReplicaIdentityFullTableWithToastedValue(PostgresConnectorConfig.SchemaRefreshMode mode, boolean tablesBeforeStart) throws Exception{
+        PostgresConnectorConfig config = new PostgresConnectorConfig(TestHelper.defaultConfig()
+                .with(PostgresConnectorConfig.SCHEMA_REFRESH_MODE, mode)
+                .build());
+        setupRecordsProducer(config);
+
+        if (tablesBeforeStart) {
+            TestHelper.execute(
+                    "DROP TABLE IF EXISTS test_table;",
+                    "CREATE TABLE test_table (id SERIAL, not_toast int, text TEXT);",
+                    "ALTER TABLE test_table REPLICA IDENTITY FULL"
+            );
+        }
+
+        if (!tablesBeforeStart) {
+            TestHelper.execute(
```

Comment:
Aren't these the same statements as in the block above? And I think this should be using `else` instead of the negated `if` for sake of comprehensibility.

Reply:
@gunnarmorling Thanks for soptting, the other block belongs after `recordsProducer.start`